File: Modul 4/Soal 3/PRAK403-NIM-Nama.c
#include <stdio.h>

int main(){
    int first, second;

    printf("");
    scanf("%d %d", &first, &second);

    for (int i = first, j = second; i <= second && j >= first; i++, j--){
        printf("%d %d", i, j);
        if(i < second || j > first){
            printf(" - ");
        }
    }

    for (int i = first, j = second; i >= second && j <= first; i--, j++){
        printf("%d %d", i, j);
        if (i > second || j < first){
            printf(" - ");
        }
    } 
}

File: Modul 4/Soal 3/PRAK403-NIM-Nama.py
first, second = map(int, input("").split())

i = first
j = second

if i < j:
    while i <= second and j >= first:
        print(f"{i} {j}", end=" ")

        if i < second or j > first:
            print("-", end=" ")
        i += 1
        j -= 1

elif i > j:
    while i >= second and j <= first:
        print(f"{i} {j}", end=" ")

        if i > second or j < first:
            print("-", end=" ")
        i -= 1
        j += 1